tools/module/generate.py

```diff
@@ -129,6 +129,8 @@ def generate_cpp(parts):
 
         #include "extension/Configuration.hpp"
 
+        #include "utility/support/yaml_LogLevel.hpp"
```

Comment:
I think this should be included within `extension/Configuration.hpp` so that you don't need to include something else.
This is also what I was talking about with yaml_expression. That should probably be included in Configuration.hpp since it's such a common usage. (but that's another PR)